src/client/mod.rs

```diff
@@ -82,47 +82,30 @@ fn read_config(p: &Path) -> SshConfig {
 
 fn connect(host: &str, params: &HostParams) {
     // Resolve host
-    let host = match params.host_name.as_deref() {
-        Some(h) => h,
-        None => host,
-    };
-    let port = match params.port {
-        None => 22,
-        Some(p) => p,
-    };
-    let host = match host.contains(':') {
-        true => host.to_string(),
-        false => format!("{}:{}", host, port),
+    let host = params.host_name.as_deref().unwrap_or(host);
+    let port = params.port.unwrap_or(22);
```

Comment:
oh, oops, can you change this to 23.

cpu uses the old telnet port.

I wonder if I should not ask ICANN for it :-)

Or should we use the Plan 9 standard -- tcp=ncpu port=17010

?

Reply:
let's go with 23 for now. That matches the Go code.

Reply:
ok done